SessionUtilitiesKit/General/SessionId.swift

```diff
@@ -7,11 +7,12 @@ public struct SessionId: Equatable, Hashable, CustomStringConvertible {
     public static let invalid: SessionId = SessionId(.standard, publicKey: [])
     
     public enum Prefix: String, CaseIterable, Hashable {
-        case standard = "05"    // Used for identified users, open groups, etc.
-        case blinded15 = "15"   // Used for authentication and participants in open groups with blinding enabled
-        case blinded25 = "25"   // Used for authentication and participants in open groups with blinding enabled
-        case unblinded = "00"   // Used for authentication in open groups with blinding disabled
-        case group = "03"       // Used for update group conversations
+        case standard = "05"            // Used for identified users, open groups, etc.
+        case blinded15 = "15"           // Used for authentication and participants in open groups with blinding enabled
+        case blinded25 = "25"           // Used for authentication and participants in open groups with blinding enabled
+        case unblinded = "00"           // Used for authentication in open groups with blinding disabled
+        case group = "03"               // Used for update group conversations
+        case versionBlinded07 = "07"    // Used for validation
```

Comment:
Just wondering whether the description for this should be something like `Used for authentication with the file and session network servers`? (just so we don't forget when looking back at it in a few years 😅)